ext/standard/string.c

```diff
@@ -1462,7 +1462,7 @@ PHPAPI zend_string *php_string_tolower(zend_string *s)
 	unsigned char *c;
 	const unsigned char *e;
 
-	if (EXPECTED(!BG(locale_changed))) {
+	if (EXPECTED(!BG(ctype_string))) {
```

Comment:
Unrelated: What's the reasoning for performing this operation for strtolower() but not strtoupper()?

Reply:
Because there is no zend_string_toupper :) We only perform ASCII lowercase internally.

Reply:
Makes sense